src/sidebar/store/modules/selection.js

```diff
@@ -64,6 +64,29 @@ function freeze(selection) {
   }
 }
 
+const setTab = (selectedTab, newTab) => {
+  // Do nothing if the "new tab" is not a valid tab.
+  if (
+    [
+      uiConstants.TAB_ANNOTATIONS,
+      uiConstants.TAB_NOTES,
+      uiConstants.TAB_ORPHANS,
+    ].indexOf(newTab) === -1
+  ) {
+    return {};
+  }
+  // Shortcut if the tab is already correct, to avoid resetting the sortKey
+  // unnecessarily.
+  if (selectedTab === newTab) {
+    return {};
+  }
+  return {
+    selectedTab: newTab,
+    sortKey: TAB_SORTKEY_DEFAULT[newTab],
+    sortKeysAvailable: TAB_SORTKEYS_AVAILABLE[newTab],
```

Comment:
`sortKeysAvailable` could be refactored into a selector in future, since it can always be determined from the selected tab and doesn't need to be maintained as a separate piece of state.